docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+services:
+  php:
+    build: .
+    volumes:
+      - ./src:/code/src
+      - ./tests:/code/tests
+      - ./phpunit.xml.dist:/code/phpunit.xml.dist
+    command: tail -F anything
+    depends_on:
+      mysql:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+  mysql:
+    image: mysql:9
+    ports:
+      - "3306:3306"
+    volumes:
+      - type: tmpfs
+        target: /var/lib/mysql
+    environment:
+      MYSQL_DATABASE: yii
+      MYSQL_ROOT_PASSWORD: root
```

Comment:
How about unifying username/password for all databases?

Reply:
#76